webpack.prod.js

```diff
@@ -0,0 +1,18 @@
+const webpack = require('webpack');
+const { merge } = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+  output: {
```

Comment:
This is a library, why do we need to bundle it with Webpack?

Reply:
Firstly, can you tell me why not?

Reply:
because when you require it in another code (for example, our SPA), it will be bundled using Webpack of that SPA.

Also, because we don't need to create a separated minified version to use in the browser (like <script src="esn-api-client.min.js></script>), so basically the Webpack script of this module will never be used.

Reply:
just like module https://github.com/OpenPaaS-Suite/esn-frontend-videoconference-calendar, it is only required in other projects, so no Webpack needed